system/system_error_monitor/config/system_error_monitor.planning_simulation.param.yaml

```diff
@@ -40,6 +40,7 @@
         /autoware/system/service_log_checker: { sf_at: "warn", lf_at: "none", spf_at: "none" }
         /autoware/system/duplicated_node_checker: default
         # /autoware/system/resource_monitoring: { sf_at: "warn", lf_at: "error", spf_at: "none" }
+        # /autoware/system/duplicated_node_checker: default
```

Comment:
What is the reason for adding this commented line in this file?

Reply:
It's to make it consistent with the config file on the launcher side, but the change is not necessary.
file:
https://github.com/autowarefoundation/autoware_launch/blob/13b28551b50141bd887042063e00cfc987c19e76/autoware_launch/config/system/system_error_monitor/system_error_monitor.planning_simulation.param.yaml#L42
FYI:
changed  by following PR:
https://github.com/autowarefoundation/autoware_launch/pull/649